Doc/library/sys.rst

```diff
@@ -1475,6 +1484,10 @@ always available.
    If there is any error occurred in the trace function, it will be unset, just
    like ``settrace(None)`` is called.
 
+   .. note::
+      Tracing is disabled while calling the trace function (e.g. a function set by
+      :func:`settrace` or :func:`setprofile`). For recursive tracing see :func:`call_tracing`.
```

Comment:
```suggestion
      :func:`!settrace`). For recursive tracing see :func:`call_tracing`.
```

* I suggest only mentioning the specific function in question here
* Avoid a cross-reference to itself